SWLOR.Game.Server/Feature/LootTableDefinition/HutlarLootTableDefinition.cs

```diff
@@ -134,5 +135,125 @@ private void QionTigers()
                 .IsRare()
                 .AddItem("q_tiger_paw", 2, 1, true);
         }
+        private void QionHiveLoot()
+        {
+            _builder.Create("QIONHIVE_RESOURCES")
+                .AddItem("ref_arkoxit", 1, 1, true)
+                .AddItem("ref_jasioclase", 20)
+                .AddItem("ref_gostian", 20)
+                .AddItem("elec_high", 20)
+                .AddItem("lth_high", 20)
+                .AddItem("emerald", 10)
+                .AddItem("fiberp_high", 20)
+                .AddItem("hyphae_wood", 20);
+
+            _builder.Create("QIONHIVE_GEAR")
+                .AddItem("dhar005s", 1)
+                .AddItem("dlar005s", 1)
+                .AddItem("dhhl005s", 1)
+                .AddItem("dlhl005s", 1)
+                .AddItem("dhbe005s", 1)
+                .AddItem("dlbe005s", 1)
+                .AddItem("dhlg005s", 1)
+                .AddItem("dllg005s", 1)
+                .AddItem("dhbr005s", 1)
+                .AddItem("dhcl005s", 1)
+                .AddItem("dlcl005s", 1)
+                .AddItem("dlbr005s", 1)
+                .AddItem("dhnk005s", 1)
+                .AddItem("dlnk005s", 1)
+                .AddItem("dhrg005s", 1)
+                .AddItem("cara_armor", 1)
+                .AddItem("cara_tunic", 1)
+                .AddItem("cara_robes", 1)
+                .AddItem("cara_cap", 1)
+                .AddItem("cara_headdress", 1)
+                .AddItem("cara_helmet", 1)
+                .AddItem("cara_boots", 1)
+                .AddItem("cara_leggings", 1)
+                .AddItem("cara_shoes", 1)
+                .AddItem("cara_bracer", 1)
+                .AddItem("cara_gloves", 1)
+                .AddItem("cara_wraps", 1)
+                .AddItem("dlrg005s", 1);
+
+            _builder.Create("QIONHIVE_BROODMOTHER_RESOURCE")
+                .AddItem("chiro_shard", 1);
+
+            _builder.Create("QIONHIVE_CREDITS")
+                .AddGold(500, 1);
+
+            _builder.Create("QIONHIVE_CHAMPION_KEY")
+                .AddItem("qion_championkey", 1);
+
+            _builder.Create("QIONHIVE_SHAMAN_KEY")
+                .AddItem("qion_shamankey", 1);
```

Comment:
What is the purpose of this key? If it's to open the door, does the door auto-close and auto-lock after a set amount of time? Does the key get destroyed upon use?

Reply:
It is to open the door, yes. The door does not auto-close and auto-lock, but it DOES destroy the key. This means that the doors will relock upon reset, and keys aren't reusable, but within a reset it won't need to be reunlocked.